### ISSUE TITLE
Prometheus Webhook is out of date

### ISSUE THREAD

**Issue by tntobias** [guardian/alerta#374]
The default Prometheus webhook handler fails with HTTP 500 from the latest version of Alertmanager (ver 0.8.0).

The initial error refers to instance field which is no longer present in the Alertmanager payload and Alerta does not provide a default for:

```
return Alert(
        resource=labels.pop('exported_instance', None) or labels.pop('instance'),
```

Alerts will post after updating the code to provide an instance default but the severity does not match the submission. 

The current Alertmanager webhook payload looks like this:
```
{
  "receiver": "alerta",
  "status": "firing",
  "alerts": [
    {
      "status": "firing",
      "labels": {
        "alertname": "thing_dead"
      },
      "annotations": {
        "description": "No things have been recorded for over 10 minutes. Something terrible is happening.",
        "severity": "critical",
        "summary": "No things for over 10 minutes"
      },
      "startsAt": "2017-08-03T15:17:37.804-04:00",
      "endsAt": "0001-01-01T00:00:00Z",
      "generatorURL": "http://somehost:9090/graph?g0.expr=sum%28irate%28messages_received_total%5B5m%5D%29%29+%3D%3D+0&g0.tab=0"
    }
  ],
  "groupLabels": {
    "alertname": "thing_dead"
  },
  "commonLabels": {
    "alertname": "thing_dead"
  },
  "commonAnnotations": {
    "description": "No things have been recorded for over 10 minutes. Something terrible is happening.",
    "severity": "critical",
    "summary": "No things for over 10 minutes"
  },
  "externalURL": "http://somehost:9093",
  "version": "4",
  "groupKey": "{}:{alertname=\"thing_dead\"}"
}
```
Submitting this to Alerta generates the following:
```
{
  "attributes": {
    "externalUrl": "http://somehost:9093",
    "ip": "somehost",
    "moreInfo": "<a href=\"http://somehost:9090/graph?g0.expr=sum%28irate%28messages_received_total%5B5m%5D%29%29+%3D%3D+0&g0.tab=0\" target=\"_blank\">Prometheus Graph</a>",
    "severity": "critical"
  },
  "correlate": [],
  "createTime": "2017-08-03T19:17:37.804Z",
  "customer": null,
  "duplicateCount": 1,
  "environment": "Production",
  "event": "thing_dead",
  "group": "Prometheus",
  "history": [
    {
      "event": "thing_dead",
      "id": "9c000736-3b6c-4422-9162-10f1b2cb6620",
      "severity": "warning",
      "text": "No things have been recorded for over 10 minutes. Something terrible is happening.",
      "type": "severity",
      "updateTime": "2017-08-03T19:17:37.804Z",
      "value": "n/a"
    },
    {
      "event": "thing_dead",
      "id": "9c000736-3b6c-4422-9162-10f1b2cb6620",
      "status": "open",
      "text": "new alert status change",
      "type": "status",
      "updateTime": "2017-08-04T12:58:39.185Z"
    }
  ],
  "href": "http://somehost:8900/alert/9c000736-3b6c-4422-9162-10f1b2cb6620",
  "id": "9c000736-3b6c-4422-9162-10f1b2cb6620",
  "lastReceiveId": "b8bef95c-cbca-4269-9a06-d05804dce03c",
  "lastReceiveTime": "2017-08-04T12:58:40.651Z",
  "origin": "prometheus/-",
  "previousSeverity": "indeterminate",
  "rawData": {
    "annotations": {
      "description": "No things have been recorded for over 10 minutes. Something terrible is happening.",
      "severity": "critical",
      "summary": "No thing for over 10 minutes"
    },
    "endsAt": "0001-01-01T00:00:00Z",
    "generatorURL": "http://somehost:9090/graph?g0.expr=sum%28irate%28messages_received_total%5B5m%5D%29%29+%3D%3D+0&g0.tab=0",
    "labels": {
      "alertname": "thing_dead"
    },
    "startsAt": "2017-08-03T15:17:37.804-04:00",
    "status": "firing"
  },
  "receiveTime": "2017-08-04T12:58:39.185Z",
  "repeat": true,
  "resource": "na",
  "service": [
    ""
  ],
  "severity": "warning",
  "status": "open",
  "tags": [],
  "text": "No thing has been recorded for over 10 minutes. Something terrible is happening.",
  "timeout": 86400,
  "trendIndication": "moreSevere",
  "type": "prometheusAlert",
  "value": "n/a"
}
```

I'm happy to submit a pull request to resolve this but I'm not sure how best to maintain backwards compatibility.

**Comment by satterly** [comment]
Isn't it as simple as providing a default when `instance` label does not exist, as you suggest?
```
    return Alert(
        resource=labels.pop('exported_instance', None) or labels.pop('instance', DEFAULT?),
```

**Comment by satterly** [comment]
Fixed by #375 ?
